sprint6/zadanie1.js

```diff
@@ -0,0 +1,23 @@
+/*
+Zadanie 1. Stwórz funkcję, zadeklaruj w niej tablicę kilku liczb całkowitych,
+oblicz sumę liczb z tablicy i wypisz do konsoli tę sumę.
+*/
+
+  function sumNumber() {
+    let arr = [1,2,3,4,5,6,7,8,9,10];
+    let suma = 0;
+    for (let i = 0; i < arr.length; i++) {
+      suma = suma + arr[i];
+    }
+    console.log(suma);
+  }
+
+/*
+Stworzyłam funkcję o nazwie "sumNumber".
+W funkcji zadeklarowałam zmienne o nazwie arr i sum.
+Zmienna "arr", jest przypisywana tablicę liczb naturalnych,
+następnie stworzyłam zmienną lokalną o nazwie "sum",
```

Comment:
`Pętla wykonuje się przy kazdej iteracji` - To zdanie jest niepotrzebne, wystarczy zdanie: `W każdej iteracji pętli wykonywane jest ...`.